ce/artifacts/activation.ts

```diff
@@ -2,13 +2,67 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-import { delimiter } from 'path';
+import { delimiter, extname } from 'path';
+import { undo as undoVariableName } from '../constants';
+import { i } from '../i18n';
 import { Session } from '../session';
-import { linq } from '../util/linq';
+import { Dictionary, linq } from '../util/linq';
 import { Uri } from '../util/uri';
 import { toXml } from '../util/xml';
 import { Artifact } from './artifact';
 
+
+function generateCmdScript(variables: Dictionary<string>, aliases: Dictionary<string>): string {
```

Comment:
What are these doing? Wasn't a non-goal of ce to not run scripts?

Reply:
Ah, are these implementing the `aliases`?

Reply:
We create scripts -- earlier it was only doing environment variables, but now we have more content (aliases)